mne/gui/_viewer.py

```diff
@@ -122,20 +126,11 @@ class Object(HasPrivateTraits):
     scene = Instance(MlabSceneModel, ())
     src = Instance(VTKDataSource)
 
-    color = Color()
-    rgbcolor = Property(depends_on='color')
+    color = Tuple()
```

Comment:
Yep, that helped me track it down. Initially I did `Instance(tuple)`, which worked but was unsatisfying. Then I learned there is an `RGBColor` class, which dropped in perfectly. Even the `veiw options` panel works properly now.